### PR TITLE
Dev/polymer1.2.3

### PR DESCRIPTION
**Pull request by anthonykoerber**

### CODE REVIEW

File: src/mm-dialog/doc.json [hunk 1]
```diff
@@ -65,5 +65,8 @@
 			"description":"Hides the dialog."
 		}
 	],
-	"behaviors": [ "stackable" ]
+	"behaviors": [ 
+		"stackable", 
+		"resolvable"
```
Comment (dlasky) [inline src/mm-dialog/doc.json]: should that be refable?

File: src/mm-group/mm-group.js [hunk 2]
```diff
@@ -108,6 +99,12 @@
 			}
 		},
 
+		detached: function() {
+			console.log("detached", this);
```
Comment (dlasky) [inline src/mm-group/mm-group.js]: womp womp

File: src/shared/behaviors/templatecomponentizable.html [hunk 3]
```diff
@@ -0,0 +1,306 @@
...
+
+				return all;
+			}, []);
+
+			Polymer.StyleUtil.clearStyleRules(style);
+
+			style.textContent += "\n\n"+Polymer.StyleUtil.toCssText(css, null, 0|preserveProperties);
+		}
+
+		function lightDomify () {
```
Comment (dlasky) [inline src/shared/behaviors/templatecomponentizable.html]: maybe worth refactoring this to be lightDomify(ele,root) ?

**Comment by dlasky** [comment]
+1 pending the 2 main things

File: src/shared/behaviors/refable.html [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+ * Copyright (c) 2015 MediaMath Inc. All rights reserved.
+ * This code may only be used under the BSD style license found at http://mediamath.github.io/strand/LICENSE.txt
+
+*/
+(function (scope) {
+
+	var Refable = {
+
+		properties: {
+			ref: {
```
Comment (shuwen) [inline src/shared/behaviors/refable.html]: should this be readOnly?

Comment (anthonykoerber) [inline src/shared/behaviors/refable.html]: probably, yes

**Pull request merged**